src/Cuda/xpmclient.cpp

```diff
@@ -673,15 +673,15 @@ void PrimeMiner::Mining(GetBlockTemplateContext* gbp, SubmitContext* submit) {
         
         multi = candi.index;
         multi <<= candi.origin;
-        chainorg = hash.shash;
-        chainorg *= multi;
+        nOrigin = hash.shash;
+        nOrigin *= multi;
         
-        testParams.nCandidateType = candi.type;
-        bool isblock = ProbablePrimeChainTestFastCuda(chainorg, testParams, mDepth);
+        testParams.nCandidateType = candi.type+1;

```

Comment:
testParams.nCandidateType = candi.type+1; // nCandidateType must follow chain type convention of node